GitUI/UserControls/FileStatusList.cs

```diff
@@ -1146,18 +1151,18 @@ private void FileStatusListView_ContextMenu_Opening(object sender, CancelEventAr
 
             if (!cm.Items.Find(_openSubmoduleMenuItem.Name, true).Any())
             {
-                cm.Items.Insert(1, _openSubmoduleMenuItem);
+                cm.Items.Insert(0, _openSubmoduleMenuItem);
             }
 
             bool isSubmoduleSelected = SelectedItem != null && SelectedItem.IsSubmodule;
 
             _openSubmoduleMenuItem.Visible = isSubmoduleSelected;
 
-            if (isSubmoduleSelected)
+            if (isSubmoduleSelected && !DisableSubmoduleMenuItemBold)
             {
-                _openSubmoduleMenuItem.Font = AppSettings.OpenSubmoduleDiffInSeparateWindow ?
-                    new Font(_openSubmoduleMenuItem.Font, FontStyle.Bold) :
-                    new Font(_openSubmoduleMenuItem.Font, FontStyle.Regular);
+                _openSubmoduleMenuItem.Font = AppSettings.OpenSubmoduleDiffInSeparateWindow
+                    ? new Font(_openSubmoduleMenuItem.Font, FontStyle.Bold)
+                    : new Font(_openSubmoduleMenuItem.Font, FontStyle.Regular);
```

Comment:
I'm not sure this is correct.

This needs to happen if a submodule is selected, irrespective of "bold or not".
The second part - i.e. set font weight - depends on two conditions now - `AppSettings.OpenSubmoduleDiffInSeparateWindow` and `DisableSubmoduleMenuItemBold`.

Reply:
This was not correct before. See the screenshot for FormCommit Index 
![image](https://user-images.githubusercontent.com/6248932/78968798-c053ed00-7b05-11ea-9785-5357c01b8ff4.png)

This worked for FormCommit worktree as FormCommit created a separate menu item with the same name was created.

See below as well.